### DISCUSSION TITLE
Rome: conjunto de ferramentas JavaScript by Facebook

### DISCUSSION THREAD

**Discussion by tarciozemel** [frontendbr/forum#1739] [Dicas]
Senhores, [coloquem vossas barbas de molho](https://www.infoq.com/news/2020/03/rome-experimental-js-toolchain/).

A coisa está em fase embrionária, mas, como consta no [site oficial](https://romejs.dev/), trata-se de

> compiler, linter, formatter, bundler, testing framework and more

◕‿◕

**Comment by ninetails** [comment]
Esqueceu do "sem dependência externa"... e feito pelo mesmo cara que criou Yarn e Babel.

Meu medo é só dele ser opinativo demais. Quando tentei rodar, _nada funciona feijoada_ (não to criticando a ferramenta, e sim avisando pra quem for ver ainda tá, como o OP disse, bem "embrionário".
